pulse_actions/handlers/treeherder_buildbot.py

```diff
@@ -13,31 +14,27 @@
 def on_buildbot_event(data, message, dry_run):
     """Act upon buildbot events."""
     # Pulse gives us a job_id and a job_guid, we need request_id.
+    treeherder_client = TreeherderClient()
     repo_name = data['project']
     job_id = data['job_id']
-    request_id = get_request_id_from_job_id(repo_name, job_id)
+    result = treeherder_client.get_jobs(repo_name, id=job_id)[0]
+    buildername = result["ref_data_name"]
+    resultset_id = result["result_set_id"]
+    revision = treeherder_client.get_resultsets(repo_name, id=resultset_id)[0]["revision"]
     action = data['action']
     status = None
 
-    # Re-trigger action
-    if action == 'retrigger':
-        make_retrigger_request(repo_name, request_id, dry_run=dry_run)
+    # Backfill action
+    if action == "backfill":
+        manual_backfill(revision, buildername, max_revisions=20, dry_run=dry_run)
```

Comment:
We are using max_revisions=5 in automatic backfilling because after Whistler seta is not supposed to coalesce more then 5 revisions in a row.